shell/edit/networking.k8s.io.ingress/Rule.vue

```diff
@@ -41,30 +50,55 @@ export default {
       } else {
         delete out.host;
       }
-      this.$emit('input', out);
+
+      this.$nextTick(() => {
+        if ((this.paths?.length === 1 && this.pathObjectIsEmpty(this.paths[0])) || this.path?.length === 0) {
```

Comment:
For the condition - `this.path?.length === 0` - should it be `this.paths?.length === 0`?

Reply:
worst kind of broken, the kind that looks like it works... fixed!